### DISCUSSION TITLE
Chore Structure Suggestions

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3770] [Ideas]
Wanting to start a public discussion about a **possible** change to the chore structure. Possible, not definite (for next or future season)

- Players get 5 chores in varying difficulty displayed at once, as is currently.
- All chores for the week are available to be loaded into the player's chore list once they complete current ones.
- If a player completes a chore, they get a new one instantly.

EG.
Players receives 5 chores today. They complete 2 of them. They would receive 2 new chores to replace the ones completed. The cap would be weekly (5 chores daily, 7 days, 35 max chores completed for the week)

Additional clarifications added by tourist:
- at weekly reset, all chores expire / are removed
- at weekly reset, 35 new chores are made available but only 5 are visible
- when completing a visible chore, another chore unlocks (of the 35) until all 35 have been completed*

This would relieve some of the daily pressure and give players a week to complete chores. What are your thoughts, criticism or suggestions to a system as such?

**SUGGESTION:
- Add a 12-24h 'preview' before weekly reset
- Ensure the VERY FIRST hard chore is something that players can start preparing BEFORE the reset (during the 12-24h preview) as to help the timezone issue

**Comment by brian-man** [comment]
`+` significantly improves the timezone bias problems
👍

**Comment by Dionis404** [comment]
The idea is good but won't imbalance appear before the auctions? 
Many pumped farms will be easier and faster to perform tasks and get all the tickets at the beginning of the week and be higher in the list of participants before the auction, which let's say on Wednesday.

**Reply by TheDcol** [reply]
A fair point, what would you suggest to counter this?

Could auctions occur at the end of the week? say Friday-Sunday?

**Reply by Dionis404** [reply]
The idea of doing the auction on the weekend is a good one. 

Then maybe we should consider the option that the tasks will be given from Monday to Friday. 
And the weekend will be without assignments. 
But in this case, players who have not had time to complete the tasks for 5 days can have time to finish them on the weekend. 
After that the list will be completely updated itself and the player will not have the opportunity to finish the task of last week.

**Reply by TheDcol** [reply]
> And the weekend will be without assignments.

I had mentioned having a day off on Discord (Sunday) so making it the weekend could work. 

Preview chores on Sunday, Monday-Friday Chores, Late Saturday/Sunday Auctions

**Comment by adamhannigan** [comment]
Awesome ideas everyone! 👏 @TheDcol @Dionis404 

The team are currently implementing the Faction Chore mechanic and taking on these suggestions ❤️
